Makefile

```diff
@@ -34,11 +34,10 @@ help:
 	@echo "Internationalization"
 	@echo "--------------------"
 	@echo ""
-	@echo "translation-django-makemessages: creates source messages for django"
-	@echo "translation-django-compilemessages: compiles all language translation sources"
-	@echo "translation-crowdin-install: installs the CrowdIn CLI"
-	@echo "translation-crowdin-upload branch=<crowdin-branch>: uploads kolibri translation sources via CrowdIn"
-	@echo "translation-crowdin-download branch=<crowdin-branch>: downloads kolibri translated languages via CrowdIn"
+	@echo "translation-extract: extract strings from application"
```

Comment:
The naming was on purpose, I think the developer needs awareness of what these commands are actually doing?

Reply:
Note that the previous naming (using the word 'django') was inaccurate because the extract command works on both front- and back-end translation strings

Reply:
Ah I see, I was referring to the command being provided by Django :) As in `makemessages` and `compilemessages` are both Django management commands.